sources/ClangSharp/Cursors/Decls/NamespaceAliasDecl.cs

```diff
@@ -0,0 +1,11 @@
+using ClangSharp.Interop;
+
+namespace ClangSharp
+{
+    public sealed class NamespaceAliasDecl: NamedDecl, IRedeclarable<NamespaceDecl>
```

Comment:
nit: There should be a space before the colon

Reply:
As above, feel free to insert it.